content/pages/about/technology-overview.md

```diff
@@ -220,12 +220,12 @@ __Server Components__
 
 __Clients__
 
-- [Gajim](http://gajim.org/)
+- [Gajim](https://gajim.org/)
 - [Psi](http://psi-im.org/)
 
 __3.4 Libraries__
 
-- [strophe](http://code.stanziq.com/strophe/) (C or JavaScript)
+- [strophe](https://code.stanziq.com/strophe/) (C or JavaScript)
```

Comment:
This link doesn't have an HTTPs version and returns a 404.

Reply:
@MilanKral could you fix the stanziq fix please?